src/Raven.Client/Documents/Indexes/IndexDefinition.cs

```diff
@@ -330,11 +330,29 @@ public void RemoveDefaultValues()
                 Fields.Remove(key);
         }
 
+        private static readonly HashSet<(string Start, string End)> _commentTokens = new HashSet<(string Start, string End)>{("//",Environment.NewLine),("/*","*/")};
```

Comment:
You cannot use `Environment.NewLine` here, you have to support `\r`, `\r\n`, `\n`, etc.